Extensibility: Extend Pizza examples (#1570)
kyma-project/busola: examples/pizzas/README.md

The `pizzas` example showcases the Busola's extensibility feature.

> The `configuration` directory consists of two Custom Resource Definitions and two Config Maps that store the information about the extensibility configuration.

Reviewer: ```suggestion
The `configuration` directory consists of two Custom Resource Definitions (CRD) and two Config Maps that store the information about the extensibility configuration.
```
Since you use the abbreviation later in the text, it's good to have it introduced earlier.

The `samples.yaml` file includes the examples of the above CRDs.

## Installation

> To test the extensibility feature using the Pizza example, simply upload all the YAML files from the `configuration` directory and `samples.yaml` file to your cluster or use the following commands:

Reviewer: ```suggestion
To test the extensibility feature using the Pizza example, simply upload all the YAML files from the `configuration` directory and the `samples.yaml` file to your cluster or use the following commands:
```

> 1. Create a `pizzas` namespace and apply the configmaps and CRDs:

Reviewer: ```suggestion
1. Create a `pizzas` namespace and apply the Config Maps and CRDs:
```

> 1. Create a `pizzas` namespace and apply the Config Maps and CRDs:

Reviewer: ```suggestion
1. Create a `pizzas` Namespace and apply the Config Maps and CRDs:
```

```bash
kubectl create namespace pizzas && kubectl apply -f configuration/ --recursive
```

> 2. Apply the samples file:

Reviewer: ```suggestion
2. Apply the `samples.yaml` file:
```
